Orange/base.py

```diff
@@ -49,11 +53,7 @@ def __call__(self, data):
 
         self.domain = data.domain
 
-        if type(self).fit is Learner.fit:
-            model = self.fit_storage(data)
-        else:
-            X, Y, W = data.X, data.Y, data.W if data.has_weights() else None
-            model = self.fit(X, Y, W)
+        model = self.fit_storage(data)
```

Comment:
IIRC, the if was there to allow having both `fit` and `fit_storage` defined in which case the more specific (numpy optimized) `fit` would be used.
Is there a reason this was removed?